.github/workflows/ci.yml

```diff
@@ -586,6 +586,7 @@ jobs:
           - test_name: lit-di-evm-identity-test
           - test_name: lit-parentchain-nonce
           - test_name: lit-ii-batch-test
+          - test_name: lit-test-stress-script
```

Comment:
Are we planning to trigger it for each CI run?

Reply:
Yes, to make sure that it doesn't go stale with any future changes (so that it's ready to go whenever we need it again). i.e. it's a test _of_ the stress logic rather than a stress test of the logic :)

The test starts three threads, each thread does two iterations; in total it shouldn't be a heavy burden on CI; I think :thinking: 

Then again, what's probably more expensive is just to start the container for the test, so maybe it could be grouped with some other step?

Reply:
/cc @Verin1005 to address your [comment](https://github.com/litentry/litentry-parachain/pull/2054#pullrequestreview-1594773483); I don't think it should add much more maintenance overhead beyond keeping the _regular_ tests up to date. There is some duplication right now but I'm planning to tackle it with #1878 and #1153

Reply:
It's fine as long as it's not running extremely long 😎